cp-attendance-hybrid/src/modules/profile/profile.page.tsx

```diff
@@ -51,7 +51,7 @@ export const Profile: React.FC<ProfileScreenProps> = ({ navigation }) => {
       <ProfileAreaViewStyled>
         <ProfileButtonAreaStyled>
           <ButtonCard icon="lock-closed-outline" text="Alterar Senha" onTap={() => console.log('click')} />
-          <ButtonCard icon="notifications-outline" text="Notificações" onTap={() => console.log('click')} />
+          {/* <ButtonCard icon="notifications-outline" text="Notificações" onTap={() => console.log('click')} /> */}
```

Comment:
Tem algum motivo pra estar comentado e não deletado? Se for o caso deixaria um TODO pra indicar qnd descomentar/deletar/sei lá